### PR TITLE
#19034 latency histogram update

### PR DESCRIPTION
**Pull request by ddhodge**
#19034

@netlify /preview/explore/query-1-performance/pg-stat-statements/

### CODE REVIEW

File: docs/content/preview/explore/query-1-performance/pg-stat-statements.md [hunk 1]
```diff
@@ -142,7 +141,105 @@ Time Span (ms) | Resolution (ms)
...
+409.6-819.2 | 25.6
+819.2-1638.4 | 51.2
+... |
+209715.2-419430.4 (3.49-6.99 min) | 13107.2
+419430.4-838860.8 (6.99-13.98 min) | 26214.4
+838860.8-1677721.6 (13.98-27.96 min) | 52428.8
+
+### Show latency percentiles
+
+Use the `yb_get_percentile` function to show percentiles for a given histogram. This function takes a JSONB histogram (including `yb_latency_histogram`) and a target percentile (as an integer) as input.
```
Comment (sushantrmishra) [inline docs/content/preview/explore/query-1-performance/pg-stat-statements.md:172]: as an integer --> this is a double precision

File: docs/content/preview/explore/query-1-performance/pg-stat-statements.md [hunk 2]
```diff
@@ -142,7 +141,105 @@ Time Span (ms) | Resolution (ms)
...
+
+```sql
+\df yb_get_percentile
+```
+
+```output
+                                         List of functions
```
Comment (sushantrmishra) [inline docs/content/preview/explore/query-1-performance/pg-stat-statements.md:179]: Remove -> List of Functions (does not add anything)

File: docs/content/preview/explore/query-1-performance/pg-stat-statements.md [hunk 3]
```diff
@@ -142,7 +141,105 @@ Time Span (ms) | Resolution (ms)
...
+
+```output
+ yb_get_percentile 
+-------------------
+               1.2
+(1 row)
+```
+
+```sql
+SELECT query, calls, total_time, min_time, max_time, mean_time, rows,
```
Comment (sushantrmishra) [inline docs/content/preview/explore/query-1-performance/pg-stat-statements.md:222]: Can we put a description similar to following: 
Display the P99, P95 and P90  latency by augmenting the pg_stat_statements output for specific query.

**Review by sushantrmishra — APPROVED** [review]

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**